website/content/maintainers/github/templates/_index.md

```diff
@@ -38,14 +39,15 @@ when you view the markdown file in GitHub unless you view the raw text.
 * [CODE_OF_CONDUCT.md](https://github.com/cncf/project-template/blob/main/CODE_OF_CONDUCT.md)
 * [SECURITY.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/SECURITY.md)
 
-## Recommended Files
+## Optional Files
 * [CONTRIBUTOR_LADDER.md](https://github.com/cncf/project-template/blob/main/CONTRIBUTOR_LADDER.md)
 * [GOVERNANCE.md](https://github.com/cncf/project-template/blob/main/GOVERNANCE.md)
 * [SECURITY_CONTACTS.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/SECURITY_CONTACTS.md)
 * [ISSUE_TEMPLATE.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/ISSUE_TEMPLATE.md)
 * [embargo-policy.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/embargo-policy.md)
 * [embargo.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/embargo.md)
 * [incident-response.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/incident-response.md)
+* [REVIEWING.md](optional/reviewing/)
 
 [contrib-strat]: https://github.com/cncf/tag-contributor-strategy/blob/main/README.md
 [project template]: https://github.com/cncf/project-template
```

Comment:
Recommend reverting this back to recommended and potentially breaking this out further into Recommended and Additional.  With a lot of the move towards more security forward awareness within projects, I'd personally like to see security templates as recommended.

Below is a potential breakout option:
```suggestion
## Recommended Files
* [GOVERNANCE.md](https://github.com/cncf/project-template/blob/main/GOVERNANCE.md)
* [ISSUE_TEMPLATE.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/ISSUE_TEMPLATE.md)
* [SECURITY_CONTACTS.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/SECURITY_CONTACTS.md)

## Additional Files
* [CONTRIBUTOR_LADDER.md](https://github.com/cncf/project-template/blob/main/CONTRIBUTOR_LADDER.md)
* [embargo.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/embargo.md)
* [embargo-policy.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/embargo-policy.md)
* [incident-response.md](https://github.com/cncf/tag-security/blob/main/project-resources/templates/incident-response.md)
* [REVIEWING.md](optional/reviewing/)

[project template]: https://github.com/cncf/project-template
[contrib-strat]: https://github.com/cncf/tag-contributor-strategy/blob/main/README.md

```

Reply:
Since the howto files clearly call out that they aren't required, how about we go back to recommended but not split out into "additional"? Just have "required' and "recommended".

Reply:
Sgtm